.cargo/config

```diff
@@ -2,4 +2,5 @@
 wasm = "build --release --target wasm32-unknown-unknown"
 wasm-debug = "build --target wasm32-unknown-unknown"
 t = "test --workspace --exclude integration_tests"
-unit-test = "test --workspace --exclude integration_tests"
\ No newline at end of file
+unit-test = "test --workspace --exclude integration_tests"
+integration-test = "test --package integration_tests -- --ignored --test-threads 1"
```

Comment:
I decided to just ignore all integration tests, so that people who run `cargo test` from workspace root wont run them.

While the integration tests are a part of the workspace, I think this is the best approach for now.

Reply:
nice!! thank you!